app/jobs/send_invitations_job.rb

```diff
@@ -64,74 +60,11 @@ def completed_some?(response)
-    return false unless Time.zone.now > Time.new(2018, 5, 10, 9).in_time_zone &&
-                        Time.zone.now < Time.new(2018, 5, 14).in_time_zone
-    person = response.protocol_subscription.person
-    # the following does not work for student voormeting because it is used in multiple measurements
-    measurement = Questionnaire.find_by_name('voormeting mentoren')&.measurements&.first
-    if measurement.blank?
-      Rails.logger.info '[Attention] ERROR: voormeting mentoren measurement not found'
-      puts 'ERROR: voormeting mentoren measurement not found'
-      return false
-    end
-    Response.where('completed_at IS NOT NULL AND completed_at >= :begin_at AND completed_at <= :end_at',
-                   begin_at: Time.new(2018, 5, 3, 10).in_time_zone,
-                   end_at: Time.new(2018, 5, 10, 9, 59).in_time_zone).where(measurement_id: measurement.id,
-                                                                            filled_out_by_id: person.id).count.positive?
-  end
-  # rubocop:enable Metrics/AbcSize
-
-  def one_time_reminder_window
-    Time.zone.now > Time.new(2018, 5, 17, 9).in_time_zone &&
-      Time.zone.now < Time.new(2018, 5, 21).in_time_zone
-  end
-
-  # rubocop:disable Metrics/CyclomaticComplexity
-  # rubocop:disable Metrics/PerceivedComplexity
-  # rubocop:disable Metrics/MethodLength
-  # rubocop:disable Metrics/AbcSize
   def mentor_texts(response)
-    if one_time_reminder_window
-      "Hoi #{target_first_name(response)}, tot morgen 18.00 uur kan je de vragenlijst " \
-      'weer voor jouw jongeren invullen. Veel succes!'
-    elsif filled_out_mentor_voormeting_last_week?(response)
-      'Bedankt voor je inzet! Door een technische fout kreeg je vorige week een verkeerde ' \
-      '\'welkom bij het onderzoek\' sms toegestuurd, maar het onderzoek loopt gewoon door. ' \
-      'Onze excuses voor de verwarring. Bij deze weer een link naar de wekelijkse vragenlijst:'
-    elsif open_questionnaire?(response, 'voormeting mentoren') && completed_some?(response)
-      if Time.zone.now > Time.new(2018, 5, 10, 9).in_time_zone && Time.zone.now < Time.new(2018, 5, 14).in_time_zone
-        'Bedankt voor je inzet! Door een technische fout kreeg je vorige week een verkeerde ' \
-        '\'welkom bij het onderzoek\' sms toegestuurd, maar het onderzoek loopt gewoon door. ' \
-        'Onze excuses voor de verwarring. Als je op de link klikt kom je eerst nog even bij ' \
-        'de allereerste vragenlijst (de voormeting), die had je nog niet ingevuld, daarna kom ' \
-        'je bij de wekelijkse vragenlijst.'
-      else
-        'Hartelijk dank voor je inzet! Naast de wekelijkse vragenlijst sturen we je deze ' \
-        'week ook nog even de allereerste vragenlijst (de voormeting), die had je nog niet ' \
-        'ingevuld. Na het invullen hiervan kom je weer bij de wekelijkse vragenlijst.'
-      end
+    if open_questionnaire?(response, 'voormeting mentoren') && completed_some?(response)
```

Comment:
Misschien wel mooier om:
``` ruby
   def open_questionnaire?(response, questionnaire_name)
     response.protocol_subscription.person.my_open_responses.select do |resp|
       resp.measurement.questionnaire.name == questionnaire_name
     end.count.positive?
   end
```

aan te passen naar een `Person` functie:
```ruby
   def has_open_questionnaire?(questionnaire_name)
     my_open_responses.select do |resp|
       resp.measurement.questionnaire.name == questionnaire_name
     end.count.positive?
   end
```

En dan hier
``` ruby
person = response.protocol_subscription.person
person.has_open_questionnaire?(questionnaire_name)
```

Reply:
Als je dan toch bezig bent, het zelfde zou je dan kunnen doen voor `completed_some?`. Je zou een `completed_responses` in person kunnen maken. Of misschien nog beter, een link naar alle responses. Dan kun je iets doen als `person.responses.completed.positive?`. Het mooie daaraan is namelijk dat we dat dan later ook voor bijvoorbeeld de responses API makkelijk kunnen hergebruiken.

Reply:
Nice, fixed.